wordpress-mobile/WordPress-iOS
# Comment edit: enable new view

Ref: #17000 

This enables the new Comment Edit view, displayed from My Site > Comments > Comment details > Edit.

To test:
- Go to My Site > Comments > Comment details > Edit.
- Verify the new view is displayed and updates the Comment as expected.


| Old | New |
|--------|-------|
| ![Simulator Screen Shot - iPhone 12 Pro Max - 2021-09-02 at 14 48 08](https://user-images.githubusercontent.com/1816888/131914123-032c0275-410b-436f-b4e6-abb8f83a267e.png) | ![Simulator Screen Shot - iPhone 12 Pro Max - 2021-09-02 at 14 48 35](https://user-images.githubusercontent.com/1816888/131914142-31a0628a-bbbe-4e02-bc00-0017f40cb799.png) |

## Regression Notes
1. Potential unintended areas of impact
The Comment could not update correctly.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Made sure the Comment updated as expected. 😄 

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
